test/IECore/MeshAlgoTest.py

```diff
@@ -32,478 +32,12 @@
-
-		# Except for vertex ids, and facevarying data
-
-		self.assertEqual( list( meshReversed.vertexIds ), list( reversed( mesh.vertexIds ) ) )
-		self.assertEqual( list( meshReversed["uv"].data ), list( reversed( mesh["uv"].data ) ) )
-
-	def testPlane( self ) :
-
-		mesh = IECore.MeshPrimitive.createPlane( IECore.Box2f( IECore.V2f( -1 ), IECore.V2f( 1 ) ), IECore.V2i( 10 ) )
-		IECore.TriangulateOp()( input = mesh, copyInput = False )
-
-		meshReversed = mesh.copy()
-		IECore.MeshAlgo.reverseWinding( meshReversed )
-
-		evaluator = IECore.MeshPrimitiveEvaluator( mesh )
-		evaluatorReversed = IECore.MeshPrimitiveEvaluator( meshReversed )
-
-		result = evaluator.createResult()
-		resultReversed = evaluatorReversed.createResult()
-
-		for i in range( 0, 1000 ) :
-
-			p = IECore.V3f( random.uniform( -1.0, 1.0 ), random.uniform( -1.0, 1.0 ), 0 )
-			evaluator.closestPoint( p, result )
-			evaluatorReversed.closestPoint( p, resultReversed )
-
-			self.assertEqual( resultReversed.normal(), -result.normal() )
-
-			reversedUV = resultReversed.vec2PrimVar( meshReversed["uv"] )
-			uv = result.vec2PrimVar( mesh["uv"] )
-			self.assertAlmostEqual( reversedUV[0], uv[0], delta = 0.0001 )
-			self.assertAlmostEqual( reversedUV[1], uv[1], delta = 0.0001 )
-
-	def testRoundTrip( self ) :
-
-		mesh = IECore.MeshPrimitive.createPlane( IECore.Box2f( IECore.V2f( -1 ), IECore.V2f( 1 ) ), IECore.V2i( 10 ) )
-		meshReversed = mesh.copy()
-		IECore.MeshAlgo.reverseWinding( meshReversed )
-		meshReversedAgain = meshReversed.copy()
-		IECore.MeshAlgo.reverseWinding( meshReversedAgain )
-
-		self.assertEqual( mesh, meshReversedAgain )
-
-	def testUVIndices( self ) :
-
-		verticesPerFace = IECore.IntVectorData( [ 3 ] )
-		vertexIds = IECore.IntVectorData( [ 0, 1, 2 ] )
-		p = IECore.V3fVectorData( [ IECore.V3f( 0, 0, 0 ), IECore.V3f( 1, 0, 0 ), IECore.V3f( 0, 1, 0 ) ] )
-		uv = IECore.V2fVectorData( [ IECore.V2f( 0, 0 ), IECore.V2f( 1, 0 ), IECore.V2f( 0, 1 ) ] )
-		uvIndices = IECore.IntVectorData( [ 0, 1, 2 ] )
-		mesh = IECore.MeshPrimitive( verticesPerFace, vertexIds, "linear", p )
-		mesh["uv"] = IECore.PrimitiveVariable( IECore.PrimitiveVariable.Interpolation.FaceVarying, uv, uvIndices )
-
-		meshReversed = mesh.copy()
-		IECore.MeshAlgo.reverseWinding( meshReversed )
-
-		# Meshes should be identical
-
-		self.assertEqual( meshReversed.interpolation, mesh.interpolation )
-		for interpolation in IECore.PrimitiveVariable.Interpolation.values.values() :
-			self.assertEqual( meshReversed.variableSize( interpolation ), mesh.variableSize( interpolation ) )
-		self.assertEqual( mesh.keys(), meshReversed.keys() )
-		self.assertEqual( mesh["P"], meshReversed["P"] )
-
-		# UV indices should change, but UV data doesn't need to
-
-		self.assertEqual( meshReversed["uv"].data, mesh["uv"].data )
-		self.assertEqual( list( meshReversed["uv"].indices ), list( reversed( mesh["uv"].indices ) ) )
+from MeshAlgoDeleteFacesTest import MeshAlgoDeleteFacesTest
```

Comment:
To be honest, I just did it this way because it made it easy to run all the MeshAlgo tests at once, without the rest of it.

`scons testCorePython TEST_CORE_SCRIPT=test/IECore/MeshAlgoTest.py`

Reply:
OK. No biggie. Let's leave as-is until such a time as we make an IECoreTest module, then we can reconsider...